02-Logging/01-Displaying-Status-Information.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>They can be methods too, although we don't typically/haven't used this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I added a mention of methods here. We do use a callable as a custom quantity in the same notebook, so it would be best not to confuse users by stating that they can only be properties.</p>